packages/cysync-core/src/dialogs/CySync/ReleaseNotes/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { BlurOverlay } from '@cypherock/cysync-ui';
+import React, { FC } from 'react';
+
+import { ReleaseNotesDialogProvider, useReleaseNotesDialog } from './context';
+
+const ReleaseNotes: FC = () => {
+  const { currentDialog, tabs, currentTab } = useReleaseNotesDialog();
+
+  return <BlurOverlay>{tabs[currentTab]?.dialogs[currentDialog]}</BlurOverlay>;
```

Comment:
Since this dialog has only one screen, we don't need to use tabs and dialogs here

Reply:
fixed